src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -2121,6 +2121,25 @@ class Program
             End Using
         End Function
 
+        <WorkItem(14085, "https://github.com/dotnet/roslyn/issues/14085")>
+        <WpfFact, Trait(Traits.Feature, Traits.Features.Completion)>
+        Public Async Function TargetTypePreselectionExactMatching() As Task
+            Using state = TestState.CreateCSharpTestState(
+                           <Document><![CDATA[
+using System.IO.
+class Program
+{
+    void Main()
+    {
+        string path = Path$$
+    }
+}]]></Document>, extraExportedTypes:={GetType(CSharpEditorFormattingService)}.ToList())
```

Comment:
Following the chain of calls that `extraExportedTypes` gets passed through, it looks like `List(Of Type)` was used unnecessarily. It ultimately gets passed to an `IEnumerable(Of Type)`